charts/dex/values.yaml

```
@@ -208,3 +208,18 @@ tolerations: []
 # -- [Affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) configuration.
 # See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#scheduling) for details.
 affinity: {}
+
+# -- [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/design.md#servicemonitor) configuration.
+servicemonitor:
+  # Enable Prometheus Operator ServiceMonitor
```

```suggestion
  # -- Enable Prometheus ServiceMonitor
```

Maybe add the link to service monitor here.